src/lucky/tags/link_helpers.cr

```diff
@@ -1,4 +1,8 @@
 module Lucky::LinkHelpers
+  def link(text : UUID, to : Lucky::RouteHelper, attrs : Array(Symbol) = [] of Symbol, **html_options) : Nil
+    a text.to_s, merge_options(html_options, link_to_href(to)), attrs
+  end
+
   def link(text, to : Lucky::RouteHelper, attrs : Array(Symbol) = [] of Symbol, **html_options) : Nil
     a text, merge_options(html_options, link_to_href(to)), attrs
```

Comment:
I mentioned this in the description.  I am totally fine with calling to_s in the method.  I just was not sure if it would have a performance hit or something.  But the more I think about it the less I am concerned.

Reply:
Also just noticed we already do this in the Ints lol https://github.com/luckyframework/lucky/blob/master/src/charms/int64_extensions.cr#L4

Reply:
Yep using https://github.com/luckyframework/lucky/blob/dd06b41926da1a162732c4c0d3ff83c22018543e/src/charms/int64_extensions.cr#L4 AllowedInTags should do the trick here! That way we don't need to modify `link` and it should wok everywhere like in `span`, etc.